Gemfile

```diff
@@ -80,7 +79,7 @@ group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'dotenv-rails'
   gem 'factory_bot_rails'
-  gem 'rspec-rails', '4.0.2'
+  gem 'rspec-rails', '~> 6.0'
```

Comment:
Ak nepotrebujeme specianu verziu tak by som tu nedal verziu vobec. Tato tam bola lebo inak to failovalo